src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -725,6 +725,44 @@ same applies to other request types:
 | DELETE       | Yes        | delete         | 204         |
 +--------------+------------+----------------+-------------+
 
+How to use a custom API endpoint in a RESTController?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you don't have any access restriction you can use ``@Endpoint``. If you
+have set a permission scope to restrict access to your endpoints,
```

Comment:
```suggestion
have set a permission scope to restrict the access to your endpoints,
```

Reply:
Actually, "restrict access" is correct :)

Reply:
Okay, thanks.